lib/systems/include/BaseLaunchController.h

```diff
@@ -37,19 +43,28 @@ class BaseLaunchController : public virtual Controller
     int16_t init_speed_target_ = 0;
 
 public:
+    /// @brief Constructor for template launch controller
+    /// @param initial_speed_target used only in simple launch controller algorithm
+    /// @note requires one method: calc_launch_algo
     BaseLaunchController(int16_t initial_speed_target)
         : init_speed_target_(initial_speed_target)
     {
         writeout_.command = BaseControllerParams::TC_COMMAND_NO_TORQUE;
         writeout_.ready = true;
     }
 
+    /// @brief ticks launch controller to progress through launch states when conditions are met
     void tick(const SysTick_s &tick,
               const PedalsSystemData_s &pedalsData,
               const float wheel_rpms[],
               const VectornavData_s &vn_data);
     LaunchStates_e get_launch_state() { return launch_state_; }
+    /// @brief calculates how speed target, the speed the car is trying to achieve during launch, is set and/or increased during launch
```

Comment:
Thanks, this is helpful. I might be nitpicking, but I'd prefer the "the speed the car is trying to achieve during launch" to be in parenthesis rather than a comma. It seemed like you were starting a list, and I had to re-read it to understand what's going on.

Reply:
Or, rather than defining speed target here, you should specify the member variable by name.
"Updates the speed_target_ member variable according to the given VectorNav input data."
I'd also like to see this function refactored to something like "update_launch_speed_target()", as that would be **far** more descriptive than `calc_launch_algo()`.

Reply:
this might have to be a higher level in person conversation about calc launch algo, because I think the name fits, but am open to discuss changes to it